gubernator/github/models.py

```diff
@@ -51,6 +51,7 @@ class GithubWebhookRaw(ndb.Model):
     repo = ndb.StringProperty()
     number = ndb.IntegerProperty(indexed=False)
     event = ndb.StringProperty()
+    guid = ndb.StringProperty()
```

Comment:
General new-datastore-property comment: if we will never want to look up a record by `guid`, we should specify `indexed=False` on this. It seems plausible to me that we may want to do that someday, so I doubt this needs changing — just checking that you've considered it.

Reply:
Yeah, this is intentional. We don't look up records by GUIDs now, but we may want to in the future or while debugging since that is the identifier used for tracing.